boards/lunakey_pico/main.py

```diff
@@ -0,0 +1,121 @@
+import board
+
+import digitalio
+import pwmio
+import time
+
+from kb import KMKKeyboard
+from kmk.keys import KC
+from kmk.extensions.RGB import RGB
+from kmk.extensions.RGB import AnimationModes
+from kmk.modules.split import Split, SplitSide, SplitType
+from kmk.modules.layers import Layers
+from kmk.modules.modtap import ModTap
+
+led = digitalio.DigitalInOut(board.GP25)
+led.direction = digitalio.Direction.OUTPUT
+led.value = True
+
+keyboard = KMKKeyboard()
+keyboard.tap_time = 100
+
+layers_ext = Layers()
+modtap_ext = ModTap()
+
+# TODO Comment one of these on each side
+split_side = SplitSide.LEFT
+split_side = SplitSide.RIGHT
+
+data_pin = board.GP1 if split_side == SplitSide.LEFT else board.GP0
+data_pin2 = board.GP0 if split_side == SplitSide.LEFT else board.GP1
+
+split = Split(
+    split_side=split_side,
+    split_type=SplitType.UART,
+    split_flip=True,
+    data_pin=data_pin,
+    data_pin2=data_pin2
+)
+
+rgb_ext = RGB(
+    pixel_pin=board.GP6,
+    num_pixels=6,
+    animation_mode=AnimationModes.BREATHING_RAINBOW
+)
+
+keyboard.modules = [layers_ext, modtap_ext, split]
+keyboard.extensions.append(rgb_ext)
+
+if split_side == SplitSide.LEFT:
+    buzzer = pwmio.PWMOut(board.GP8, variable_frequency=True)
+    OFF = 0
+    ON = 2**15
+    buzzer.duty_cycle = ON
+    buzzer.frequency = 2000
+    time.sleep(0.2)
+    buzzer.frequency = 1000
+    time.sleep(0.2)
+    buzzer.duty_cycle = OFF
+
+_______ = KC.TRNS
+XXXXXXX = KC.NO
+
+LOWER = KC.MO(1)
+RAISE = KC.MO(2)
+ADJUST = KC.MO(3)
+CT_TAB = KC.MT(KC.TAB, KC.LCTRL)
+CT_QUOT = KC.MT(KC.QUOT, KC.LCTRL)
+SF_MINS = KC.MT(KC.MINS, KC.LSHIFT)
+SG_PSCR = KC.LSFT(KC.LGUI(KC.PSCR))
+SF_PSCR = KC.LSFT(KC.PSCR)
+CG_RGHT = KC.LCTRL(KC.LGUI(KC.RGHT))
+
+keyboard.keymap = [
```

Comment:
@kdb424 what are your thoughts on default keymaps living in board definitions like this? likewise, what are your thoughts on things like the buzzer/beep at startup as seen in L49-58 above?

Reply:
I think that as much of the definition that will be static should be in kb.py as possible. That said, there is a very specific reason for these as far as I can tell, so I would give it a pass.

Reply:
@kdb424 @klardotsh Well, I agree the idea of kbd424. I think that the default keymaps and other codes become a reference sample code set for users. And, it would be better that users can start using a keyboard without any writing a code, I guess. In addition, I think that the default code set should provide all abilities to use all features the PCB has.

Reply:
@kdb424 @klardotsh Of course, "The default code should become minimum" is also good to me. Actually, the QMK Firmware has this policy. If we choose this policy, I will be able to reduce some codes from this pull request. But, if this policy becomes as a base line, I guess that we need to define the place for the variation of keymaps and other codes to use some keyboard-specific features.